media/js/firefox/new/variation-scene1.js

```diff
@@ -8,12 +8,40 @@ For 'standard' /new variations, this file will append all query params
 so the correct template is rendered on /download/thanks/.
 */
 
-(function($) {
+(function() {
```

Comment:
This code feels like it's also now getting complicated enough that it could do with some tests?